### PR TITLE
Added timeout parameter to the execute method

### PR DESCRIPTION
**Pull request by PaulRobson85**

### CODE REVIEW

**Review by andykuszyk — APPROVED** [review]

File: simpcass/__init__.py [hunk 1]
```diff
@@ -58,9 +58,9 @@ def execute(self, cql, args=None):
         session = self._try_get_session()
         statement = SimpleStatement(cql)
         if args is None:
-            rows = session.execute(statement)
+            rows = session.execute(statement, timeout=timeout)
```
Comment (andykuszyk) [inline simpcass/__init__.py:61]: I'm not sure if the default value is `None`: instead could you say something like:
```
rows = session.execute(statement) if timeout is None else session.execute(statement, timeout=timeout)
```

**Review by andykuszyk — CHANGES_REQUESTED** [review]

File: simpcass/__init__.py (hunk 1 above)
Comment (PaulRobson85) [reply simpcass/__init__.py:61]: It seems to work happily enough, I assumed that passing None was the same as not passing the parameter at all?

Comment (PaulRobson85) [reply simpcass/__init__.py:61]: Can make the change though if you prefer.

Comment (andykuszyk) [reply simpcass/__init__.py:61]: Yeah, I'm just not sure what the default value is under the hood - if there already is a default value (not `None`) then we'd be overwriting it with `None`.

If it works as is then let's leave it :)

**Pull request merged**